include/maliput/base/distance_router.h

```diff
@@ -0,0 +1,70 @@
+// BSD 3-Clause License
+//
+// Copyright (c) 2023, Woven by Toyota. All rights reserved.
```

Comment:
nit: Ups, we are in 2024 now :smile:

Reply:
[64bf98d](https://github.com/maliput/maliput/pull/619/commits/64bf98db8ea25dea6a6293ea3cdeac15172bf872)